src/pages/meeting-rooms.tsx

```
@@ -2,26 +2,27 @@ import Drawer from "@/components/commons/Drawer";
-import pickedReservationAtom from "@/components/pages/meeting-rooms/context/pickedReservation";
+import {
+  MeetingRoomsSkeleton,
+  ReservationForm,
+  RoomSelection,
+} from "@/components/pages/meeting-rooms/components";
+import {
+  pickedDateAtom,
+  pickedReservationAtom,
+} from "@/components/pages/meeting-rooms/context";
 import { useGetReservations } from "@/components/pages/meeting-rooms/hooks/useGetReservations";
 import Error from "@/components/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)